changelogs/fragments/1780-zos_mvs_raw-verbose-fail.yml

```
@@ -0,0 +1,8 @@
+bugfixes:
+  - zos_mvs_raw - If a program failed with a non-zero return code and verbose was false, the module would succeed.
+    Whereas, if the program failed and verbose was true the module would fail.
+    Fix now has a consistent behavior and fails in both cases.
```

If we do end up removing the 'whereas...' sentence, this sentence here could be:
```suggestion
    Fix now fails the module for all instances where a program has a non-zero return code.
```